java-checks/src/main/java/org/sonar/java/checks/DefaultEncodingUsageCheck.java

```diff
@@ -125,7 +131,27 @@ private boolean checkForbiddenTypes(Tree reportTree, Type symbolType) {
       constructor(JAVA_UTIL_FORMATTER).parameters(JAVA_IO_OUTPUTSTREAM),
       constructor(JAVA_UTIL_SCANNER).parameters(JAVA_IO_FILE),
       constructor(JAVA_UTIL_SCANNER).parameters(JAVA_NIO_FILE_PATH),
-      constructor(JAVA_UTIL_SCANNER).parameters(JAVA_IO_INPUTSTREAM));
+      constructor(JAVA_UTIL_SCANNER).parameters(JAVA_IO_INPUTSTREAM),
+      method(COMMONS_IOUTILS, "copy").parameters(JAVA_IO_INPUTSTREAM, JAVA_IO_WRITER),
+      method(COMMONS_IOUTILS, "copy").parameters(JAVA_IO_READER, JAVA_IO_OUTPUTSTREAM),
+      method(COMMONS_IOUTILS, "readLines").parameters(JAVA_IO_INPUTSTREAM),
+      method(COMMONS_IOUTILS, "toByteArray").parameters(JAVA_IO_READER),
+      method(COMMONS_IOUTILS, "toByteArray").parameters(JAVA_LANG_STRING),
+      method(COMMONS_IOUTILS, "toCharArray").parameters(JAVA_IO_INPUTSTREAM),
+      method(COMMONS_IOUTILS, "toInputStream").parameters(TypeCriteria.subtypeOf(JAVA_LANG_CHARSEQUENCE)),
+      method(COMMONS_IOUTILS, "toString").parameters(BYTE_ARRAY),
+      method(COMMONS_IOUTILS, "toString").parameters("java.net.URI"),
+      method(COMMONS_IOUTILS, "toString").parameters("java.net.URL"),
+      method(COMMONS_IOUTILS, "write").parameters("char[]", JAVA_IO_OUTPUTSTREAM),
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "write" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)